provider-ui/package.json

```diff
@@ -27,9 +27,10 @@
     "styled-jsx": "latest"
   },
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "cypress": "^4.8.0",
```

Comment:
Can you please incorporate the provider-ui cypress tests in Github-Actions as done for meshery-ui ?